libs/core/src/lib/avatar-group/project.json

```diff
@@ -1,32 +1,26 @@
 {
     "name": "core-avatar-group",
     "$schema": "../../../../../node_modules/nx/schemas/project-schema.json",
-    "projectType": "library",
-    "sourceRoot": "libs/core/src/lib/avatar-group",
+    "sourceRoot": "libs/core/src/lib/avatar-group/src",
     "prefix": "fd",
+    "tags": ["scope:fd", "type:lib"],
+    "projectType": "library",
     "targets": {
         "build": {
-            "executor": "@nx/angular:ng-packagr-lite",
-            "outputs": ["{workspaceRoot}/dist/libs/core/avatar-group"],
+            "executor": "@nx/angular:package",
```

Comment:
Is there a reason executor being changed?

Reply:
Not really, just the nx lib generator incorrect config